_includes/footer.html

```diff
@@ -3,8 +3,20 @@
 <hr width="50%">
 <p class="text-muted text-center">
     <small>
-        A <u><a href="https://techworkerscoalition.org">Tech Workers Coalition</a></u> project.
-        This site is <u><a href="{{ site.links.github }}">open source</a></u>.
+        Want to contribute to this newsletter or share a story? We'd love to hear from you! Email
+        <a href="mailto:twcnewsletter@protonmail.com">twcnewsletter@protonmail.com</a>
+        or <a href="https://github.com/techworkersco/techworkersco.github.io/issues">open a new issue</a>.
+    </small>
+</p>
+
+<p class="text-center">
+    ✊🏼💛
+</p>
+
+<p class="text-muted text-center">
+    <small>
+        A <a href="https://techworkerscoalition.org">Tech Workers Coalition</a> project.
```

Comment:
what about including all of the new text here, and using `<br>` ?

i think it might look nicer without all the extra spacing.